de.dashup.model/src/test/java/de/dashup/test/DatabaseUnitTest.java

```diff
@@ -6,17 +6,15 @@
 import de.dashup.shared.User;
 import de.dashup.util.string.Hash;
 import org.json.JSONArray;
-import org.junit.jupiter.api.Assertions;
-import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.*;
```

Comment:
Do we really need a * import here?

Reply:
IntelliJ improves all imports in such a way that it is as efficient as possible.